src/components/builder/simple-conditional.stories.tsx

```diff
@@ -54,3 +56,82 @@ const Template: StoryFn<typeof SimpleConditional> = () => <SimpleConditional />;
 export const Default = {
   render: Template,
 };
+
+export const SelectATextComponent = {
+  render: Template,
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    let eqInput = canvas.queryByLabelText('Has the value');
+
+    expect(eqInput).toBeNull();
+
+    const showInput = canvas.getByLabelText('This component should display');
+    await userEvent.click(showInput);
+    await userEvent.keyboard('[ArrowDown]');
+    await userEvent.click(canvas.getByText('True'));
+
+    const whenInput = canvas.getByLabelText('When the form component');
+    await userEvent.click(whenInput);
+    await userEvent.keyboard('[ArrowDown]');
+    await userEvent.click(canvas.getByText('Textfield 1 (text1)'));
+
+    eqInput = canvas.queryByLabelText('Has the value');
+
+    expect(eqInput).not.toBeNull();
+    expect(eqInput.type).toEqual('text');
+
+    const svgTags = document.getElementsByTagName('svg');
+    // 0 is the cross for the 'This component should display' field, 1 is the arrow of the dropdown
+    const littleCross = svgTags[2];
+    await userEvent.click(littleCross);
```

Comment:
I don't know if it's possible, but a class selector might more robust?

Reply:
react-select is notoriously bad for that :/